src/flag_gems/ops/maximum.py

```diff
@@ -0,0 +1,29 @@
+import logging
+
+import torch
+import triton
+import triton.language as tl
+
+from ..utils import pointwise_dynamic
+
+
+@pointwise_dynamic(
+    is_tensor=[True, True], dtypes=[float, float], promotion_methods=[(0, 1, "DEFAULT")]
+)
+@triton.jit
+def maximum_kernel(X, Y):
+    return tl.maximum(X, Y)
+
+
+def convert_bf16_to_fp32(tensor):
+    if tensor.dtype == torch.bfloat16:
+        return tensor.to(torch.float32)
+    return tensor
```

Comment:
does not tl.maximum accept bf16? suggest casting tensor to fp32 in triton kernel.

Reply:
if the dtype is bf16, the following error occurs
![b0feace2-57e6-4ec2-96a1-728c4225ba4d](https://github.com/user-attachments/assets/b69fdbd0-9afc-4fb2-a199-46d298d3399c)